### ISSUE TITLE
Social sharing text color contrast is not accessible on crimson Spine

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/WSU-spine#408]
<img width="187" alt="screen shot 2017-06-15 at 4 51 29 pm" src="https://user-images.githubusercontent.com/286171/27206264-37ad859c-51eb-11e7-9e1f-a329b3264db6.png">

`#717171` on `#eff0f1` is 4.28:1.

The crimson Spine isn't widely used, but we can still get ahead of it.

**Comment by jeremyfelt** [comment]
Fixed for all colors in #418.
